problem_one.rb

```diff
@@ -0,0 +1,7 @@
+# Given an integer array of size N, compute the sum of all even numbers in this array.
+
+class ProblemOne
+  def sum_even(array)
+    array.select(&:even?).reduce(:+) || 0
```

Comment:
How did you came about to using "reduce" ? 
Can we avoid `|| 0` ?

Reply:
I tried looking for a more elegant approach and found one at http://ruby-doc.org/core-2.5.1/Enumerable.html#method-i-inject. Used ternary operator instead of `|| 0`: https://github.com/Agilefreaks/Test/pull/42/commits/72ffffbdcdad27b5697dde8b9673c0b635e9a3c6.